src/context/auth/authContextController/AuthContextController.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+
+import { AuthContext } from '../authContext/AuthContext';
+import { AuthContextControllerProps } from './AuthContextController.types';
+import { createCheckIsLoggedInAction } from 'api/actions/auth/authActions';
+import { useQuery } from 'api/hooks';
+import { AuthStorage } from '../authStorage.enum';
+
+export const AuthContextController = ({ children }: AuthContextControllerProps) => {
+  const [token, setToken] = useState<string | null | undefined>(localStorage.getItem(AuthStorage.TOKEN));
+
+  const { error } = useQuery<string>(createCheckIsLoggedInAction());
```

Comment:
Czy to strzela do api nawet jeśli wiadomo, że nie ma tokena, np podczas inicjalizacji?